packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [3.9.0]
```

Comment:
Let's first release the MFA brick

Reply:
The MFA PR is merged into develop, so you can update your fix branch with the latest changes